### DISCUSSION TITLE
Project abandoned? Development halted?

### DISCUSSION THREAD

**Discussion by eduardopelitti** [family/connectkit#427] [General]
Hi all, it feels like the project has in fact been abandoned now. 

Please correct me if I'm wrong but maintenance has been minimum for months, plus new issues and pull requests to include new chains or localizations are left unanswered by the dev team.

I think it may be time for us to move to other libraries, **but just wanted to check with the team for an official reply on this**.

It's been a long run for us with ConnectKit (I think we started with the first minor) with several apps in production at the moment and looking to launch at least two more in the coming months. 

**In case it's abandoned we just wanted to show our appreciation to your contribution to the open source space.**

**Comment by lochie** [comment]
Hey @eduardopelitti! This library is very much still in active development, we're just planning some updates behind the scenes to improve usability and scalability of ConnectKit to better match some future ecosystem proposals. 

We haven't pushed updates in a while as the library has been fairly stable and most improvements requested can be achieved through the API and/or via the wagmi config (e.g adding new chains).

Once we have our internal plan outlined we will be sure to share what's happening. There shouldn't be anything too unexpected as we want to provide the  same best user and developer experience possible with this library.

2025 is going to be a big year 🫡

**Comment by eduardopelitti** [comment]
Great to hear!

Will close the thread to avoid creating noise in the space.

Thanks for the quick reply :)

**Reply by lochie** [reply]
Thanks for being proactive and checking in. It's always good to hear feedback!
